packages/loader/container-loader/api-report/container-loader.legacy.alpha.api.md

```diff
@@ -70,12 +70,6 @@ export interface IFluidModuleWithDetails {
     module: IFluidModule;
 }
 
-// @alpha @deprecated (undocumented)
-export interface ILoaderOptions extends ILoaderOptions_2 {
```

Comment:
I think you need a changeset for this, you can link to the deprecation release note.